mergify_engine/tests/functional/test_debug.py

```diff
@@ -141,51 +140,15 @@ async def test_debugger(self):
  'milestone': '',
  'number': {p['number']},
  'review-requested': [],
- 'status-failure': [],
+ 'status-failure': ['Summary'],
  'status-neutral': [],
- 'status-success': ['Summary'],
+ 'status-success': [],
  'title': 'test_debugger: pull request n1 from fork'}}
 is_behind: False
-mergeable_state: clean
+mergeable_state: unstable
 * MERGIFY LAST CHECKS:
-[Summary]: success | 1 potential rule | {summary_html_url}
-> ### Rule: comment (comment)
-> - [X] `base={self.main_branch_name}`
-> - [ ] any of:
->   - [ ] `label=doubt`
->   - [ ] `label=suspect`
-> - [X] all of:
->   - [X] `number>0`
->   - [X] `title~=pull request`
-> 
-> <hr />
-> :sparkling_heart:&nbsp;&nbsp;Mergify is proud to provide this service for free to open source projects.
-> 
-> :rocket:&nbsp;&nbsp;You can help us by [becoming a sponsor](/sponsors/Mergifyio)!
-> <hr />
-> 
-> <details>
-> <summary>Mergify commands and options</summary>
-> 
-> <br />
-> 
-> More conditions and actions can be found in the [documentation](https://docs.mergify.io/).
-> 
-> You can also trigger Mergify actions by commenting on this pull request:
-> 
-> - `@Mergifyio refresh` will re-evaluate the rules
-> - `@Mergifyio rebase` will rebase this PR on its base branch
-> - `@Mergifyio update` will merge the base branch into this PR
-> - `@Mergifyio backport <destination>` will backport this PR on `<destination>` branch
-> 
-> Additionally, on Mergify [dashboard](https://dashboard.mergify.io/) you can:
-> 
-> - look at your merge queues
-> - generate the Mergify configuration with the config editor.
-> 
-> Finally, you can contact us on https://mergify.io/
-> </details>
-> <!-- J1J1bGU6IGNvbW1lbnQgKGNvbW1lbnQpJzogbmV1dHJhbAo= -->
+[Summary]: failure | Mergify is disabled | {summary_html_url}
```

Comment:
This could work, but I prefer we add the PUBLIC_REPOSITORY feature above to this test, this will check more thing on the debugger output.